templates/usr/local/kafka/config/log4j.properties.j2

```diff
@@ -22,7 +22,6 @@ log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
 log4j.appender.kafkaAppender=org.apache.log4j.RollingFileAppender
-log4j.appender.kafkaAppender.DatePattern='.'yyyy-MM-dd-HH
```

Comment:
e.g. `/var/log/kafka/server.log`:
```
...
[2016-02-12 17:46:42,003] INFO Property log.cleaner.enable is overridden to false (kafka.utils.VerifiableProperties)
[2016-02-12 17:46:42,004] INFO Property log.dirs is overridden to /tmp/kafka-logs (kafka.utils.VerifiableProperties)
[2016-02-12 17:46:42,004] INFO Property log.retention.check.interval.ms is overridden to 300000 (kafka.utils.VerifiableProperties)
[2016-02-12 17:46:42,004] INFO Property log.retention.hours is overridden to 168 (kafka.utils.VerifiableProperties)
...
```

i saw warns in `/var/log/kafka.err` (which i think is just `2>>` from the `/usr/local/kafka/bin/kafka-server-start.sh`) saying `datePattern` is not a supported property of `org.apache.log4j.RollingFileAppender`.